builder/builder.go

```diff
@@ -608,17 +655,40 @@ func getBundleSetKeysSorted(set bundleSet) []string {
 	return keys
 }
 
+// isLocalPackagePath checks if path is a local-packages definition file
+func (b *Builder) isLocalPackagePath(path string) bool {
+	return strings.HasSuffix(path, b.LocalPackagesFile)
+}
+
+// isUpstreamPackagePath checks if path is an upstream packages definition file
+func isUpstreamPackagePath(path string) bool {
+	return strings.HasSuffix(path, "/packages")
+}
+
+// isPathToPackageFile checks if the path is a local or upstream package definition file
+func (b *Builder) isPathToPackageFile(path string) bool {
+	return b.isLocalPackagePath(path) || isUpstreamPackagePath(path)
+}
+
 func (b *Builder) getBundleFromName(name string) (*bundle, error) {
+	var bundle *bundle
 	path, err := b.getBundlePath(name)
 	if err != nil {
 		return nil, err
 	}
-	bundle, err := parseBundleFile(path)
-	if err != nil {
-		return nil, err
-	}
-	if err = validateBundleFilename(path); err != nil {
-		return nil, err
+	if b.isPathToPackageFile(path) {
+		bundle, err = newBundleFromPackage(name, path)
```

Comment:
just

       return newBundleFromPackage(name, path)

then remove the else.

Reply:
Done